packages/dialect-react/components/ApiContext/index.tsx

```diff
@@ -36,27 +36,25 @@ type PropsType = {
   dapp?: string; // base58 public key format
 };
 
+export type { WalletName };
 export type WalletType = WalletContextState | AnchorWallet | null | undefined;
 export type ProgramType = anchor.Program | null;
 
-export const WalletName = { ...AdapterWalletName, Anchor: 'Anchor' as const };
-export type WalletNameType = keyof typeof WalletName;
-
-export const getWalletName = (wallet: WalletType): WalletNameType | null => {
+export const getWalletName = (wallet: WalletType): WalletName | null => {
```

Comment:
I'm guessing you've removed the WalletName enum/object in order to avoid the require of the `wallet-adapter-base`. 

If it's not much to ask, but can you, please implement some enum/object that matches that object so that non-ts project can still avoid using "magic strings"? Or is there a way to still use the WalletName enum from adapter base to implement the object?

Reply:
no, not really, that's what has been changed in wallet adapter public api, they removed this enum..
but I'll do my best to avoid magic strings

Reply:
I would use the enum you've introduced as the type for this, wdyt?